apps/nextjs-pages-router/.eslintrc.json

```diff
@@ -1,3 +1,3 @@
 {
-  "extends": ["next/babel","next/core-web-vitals"]
+  "extends": "next/core-web-vitals"
```

Comment:
Same as above, but for the Pages Router: https://nextjs.org/docs/pages/building-your-application/configuring/eslint